src/mcc_tools.jl

```diff
@@ -165,20 +165,55 @@ end
 
 
 """
-	sort_polytomies!(t1::Tree, t2::Tree, MCCs)
+	sort_polytomies!(t1::Tree, t2::Tree, MCCs; strict=false)
 
 Sort nodes of `t2` such that leaves of `t1` and `t2` in the same MCC face each other.
 In a tanglegram with nodes colored according to MCC, lines of the same color will not cross.
 The order of `t1` serves as a guide and is left unchanged.
+Note that if strict `sort_polytomies!` is applied to multiple tree pairs it may not have
```

Comment:
Is that only the case with `strict=true`?

Reply:
I believe so - the issue is that using strict resolve you may have a unresolved polytomy that is still split between mccs which leads to potential issues with ordering, however for liberal resolve all such polytomies are resolved and the issue is gone.